content/blog/the-world-of-web-rag/index.md

```diff
@@ -0,0 +1,359 @@
+---
+layout: blog-post
+title: The World of Web RAG
+description: Let's discover how to make your LLMs make a quick Google search before answering your questions
+featured_image: thumbnail.jpg
+images: ["blog/the-world-of-web-rag/thumbnail.jpg"]
+alt_image: A person water-skiing in front of the Rose Island sea platform in a vintage picture.
+toc: True
+date: 2023-11-09
+last_updated:  2023-11-09
+authors:
+  - Sara Zanzottera
+tags: ["Haystack 2.0"]
+---	
+
+In an earlier post of the Haystack 2.0 series, we've seen how to build RAG and indexing pipelines. An application that uses these two pipelines is practical if you have an extensive, private collection of documents and need to perform RAG on such data only. However, in many cases, you may want to get data from the Internet: from news outlets, documentation pages, and so on.
```

Comment:
I would rephrase because we will probably post this before the others: "An application that uses these two pipelines is practical if you have an extensive, private collection of documents and need to perform RAG on such data only. One pipeline to index documents to your own, private document store (dubbed an indexing pipeline), and another to query it with a language model (the querying pipeline). However,..."